src/metabase/server/middleware/device_cookie.clj

```diff
@@ -0,0 +1,34 @@
+(ns metabase.server.middleware.device-cookie
+  "Middleware that sets a permanent device identifier cookie so we can identify logins from new devices. This is mostly
```

Comment:
Technically, isn't this about the _user agent_ (aka web browser) instead of the device? For instance, if a person uses Metabase via two different browsers on the same laptop, that person still gets two distinct cookies.

Reply:
Yeah. That's true. I'll update the docstring